various-receipes/db-optimization/docker-compose-postgres.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.6'
+services:
+  database:
+    image: postgres:11
+    command: postgres -c fsync=off -c synchronous_commit=off -c full_page_writes=off -c random_page_cost=1.0
```

Comment:
My PR shows how to disable logging also

Reply:
what do you mean? I didn't change anything in the Postgres docker